### DISCUSSION TITLE
Verified Publisher

### DISCUSSION THREAD

**Discussion by aaronshurley** [BetaHuhn/repo-file-sync-action#235] [Ideas] [answered]
👋 Hi there! We stumbled across this action and it looks like it would fit our needs well. However, our project has a requirement that we only use GitHub Actions that (a) we create ourselves or (b) are from verified publishers. Have y'all considered becoming verified?

**Comment by BetaHuhn** [comment, marked as answer]
As far as I know there is now way to become a "verified publisher" for a GitHub Action without being a partner organisation, see the [GitHub Docs](https://docs.github.com/en/developers/github-marketplace/github-marketplace-overview/about-marketplace-badges#for-github-actions).

The [publisher verification](https://docs.github.com/en/developers/github-marketplace/github-marketplace-overview/applying-for-publisher-verification-for-your-organization) process only affects published apps afaik and can only be done for a GitHub organisation and not a single user. At the moment I don't have any plans to move this action to a organisation.
